class UserAlreadyExistsError(Exception):
    """Пользователь с таким email уже существует."""
